coordinator/core/src/main/kotlin/net/consensys/zkevm/coordinator/clients/BatchExecutionProverRequestResponse.kt

```diff
@@ -7,7 +7,7 @@ import tech.pegasys.teku.ethereum.executionclient.schema.ExecutionPayloadV1
 data class BatchExecutionProofRequestV1(
   val blocks: List<ExecutionPayloadV1>,
   val tracesResponse: GenerateTracesResponse,
-  val type2StateData: GetZkEVMStateMerkleProofResponse
+  val type2StateData: build.linea.clients.GetZkEVMStateMerkleProofResponse
```

Comment:
chore: can we do `import build.linea.clients.GetZkEVMStateMerkleProofResponse` instead of adding the package prefix, to align with `BlockInterval`